site/profile/data/common.yaml

```diff
@@ -8,7 +8,7 @@ profile::core::convenience::packages:
   - "neovim"
   - "parallel"
   - "rsync"
-  - "screen"
+#  - "screen" not available on EL8
```

Comment:
It is present in epel.  https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/s/  Maybe the tags you used prevented epel from being setup?

Reply:
Oh indeed. It was probably because of the tags.

Reply:
you're right. the screen installation still works.